pd/synth/scripts/gateAnalysis.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2021 Thales DIS design services SAS

```

Comment:
A small suggestion. I think according to PEP8 the file name (module name) should be lower case (snake_case, i.e., `gate_analysis.py`).
https://www.python.org/dev/peps/pep-0008/#package-and-module-names

I would suggest you quickly run the file through [yapf](ttps://github.com/google/yapf), so that we are sure it is PEP8 conform :-)

Reply:
Doing the different between Yapf and Black, we noticed that Black reformats better the if/than/else conditions. In Thales/Invia, we prefer Black. I have used it to reformat gate_analysis.py
BTW, should not be interesting to select a good system verilog reformatter for CVA6 project ?!?

Reply:
Yes, we should really format everything using `verible`. I was just afraid of the large diff :-D

Reply:
Black is fine by me (never used it). As long we have it more or less formatted :-D